src/index.html

```diff
@@ -1,12 +1,32 @@
+<!-- @format -->
+
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-</head>
-<body>
-    <h1>Hello webpack!</h1>
-    <p>dsadasd</p>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LeaderBoard</title>
+  </head>
+  <body>
+    <h1>Leaderboard</h1>
+    <div class="flex">
+      <div>
+        <div>
+          <h3>Recent scores</h3>
+          <button type="button" class="refresh-btn">Refresh</button>
+        </div>
+        <ul class="leader-board">
+        </ul>
+      </div>
+      <div>
+        <h3>Add your score</h3>
+        <form action="#">
+          <input type="text" placeholder="Your name" /><br>
+          <input type="text" placeholder="Your score" /><br>
+          <button type="submit">Submit</button>
+        </form>
+      </div>
+    </div>
```

Comment:
- [ ] Please, in place to use HTML code to the code, I invite you to create the project with js code, preferable inside  `src/index.js`. Try this, on this script use `<div id="root"></div>`, after that use index.js to write all this structure. 🐱 
- Don't forget to follow the ES6 best practices when you do it.
- `index.html main contain only root element`

Reply:
@vichuge I can't see this in project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md) Is it good practise create everything in js file?

Reply:
@ahmetbozaci hello there, I would like to clarify that what you did was enough for this milestone but if you created it in js would still be fine. So don't worry about it 😄

Reply:
@minipeshy Thank you.